src/index.html

```diff
@@ -9,11 +9,46 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>Product cards</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
+<body class="body">
+  <div class="goods-card" data-qa="card">
+    <div class="goods-img">
+      <img src="./images/imac.jpeg" alt="Моноблок APPLE A1419 iMac 27&#x201c; Retina 5K (MNED2UA/A)">
```

Comment:
for alt 'apple imac' would be enough, more information available in the text below,
and also for alt attribute only lowercase letters are used